main.go

```diff
@@ -841,22 +841,38 @@ func getDefaultPort(portFlag string, usbInterfaces []string) (port string, err e
 	return "", errors.New("port you specified '" + strings.Join(portCandidates, ",") + "' does not exist, available ports are " + strings.Join(ports, ", "))
 }
 
-func usage() {
-	fmt.Fprintln(os.Stderr, "TinyGo is a Go compiler for small places.")
-	fmt.Fprintln(os.Stderr, "version:", goenv.Version)
-	fmt.Fprintf(os.Stderr, "usage: %s command [-printir] [-target=<target>] -o <output> <input>\n", os.Args[0])
-	fmt.Fprintln(os.Stderr, "\ncommands:")
-	fmt.Fprintln(os.Stderr, "  build: compile packages and dependencies")
-	fmt.Fprintln(os.Stderr, "  run:   compile and run immediately")
-	fmt.Fprintln(os.Stderr, "  test:  test packages")
-	fmt.Fprintln(os.Stderr, "  flash: compile and flash to the device")
-	fmt.Fprintln(os.Stderr, "  gdb:   run/flash and immediately enter GDB")
-	fmt.Fprintln(os.Stderr, "  env:   list environment variables used during build")
-	fmt.Fprintln(os.Stderr, "  list:  run go list using the TinyGo root")
-	fmt.Fprintln(os.Stderr, "  clean: empty cache directory ("+goenv.Get("GOCACHE")+")")
-	fmt.Fprintln(os.Stderr, "  help:  print this help text")
-	fmt.Fprintln(os.Stderr, "\nflags:")
-	flag.PrintDefaults()
+func usage(command string) {
+	version := goenv.Version
+	if strings.HasSuffix(goenv.Version, "-dev") && gitSha1 != "" {
+		version += "-" + gitSha1
+	}
+
```

Comment:
What do you think of using os.Args[1] here directly to avoid changing the usage signature?

Reply:
I'm sorry, I don't understand your suggestion.
Please give me a suggestion if you can.

![image](https://user-images.githubusercontent.com/9251039/124127684-00124600-dab7-11eb-8847-1b6841049d0e.png)

Reply:
Sure, no prob.